static/lang/re-en.json

```
@@ -3411,6 +3411,9 @@
                     "MatchingSuitPrompt": "Is the drawn card's suit related the check?",
                     "WealOrWoePrompt": "Select Weal or Woe."
                 },
+                "DrakeRifle": {
+                    "CriticalSpecialization": "On a critical hit, the spittle clings to the target and they take persistent damage of the same type as the weapon equal to @Damage[(1d4 + @item.system.damage.dice)[persistent,@item.system.damage.damageType]]{1d4 + the number of weapon damage dice}."
```

```suggestion
                    "CriticalSpecialization": {
                        "Acid": "On a critical hit, the spittle clings to the target and they take @Damage[(1d4 + @item.system.damage.dice)[persistent,acid]]."
                        "Cold": "On a critical hit, ...",
                        "Electricity": "On a critical hit, ...",
                        "Fire": "On a critical hit, ...",
                        "Poison": "On a critical hit, ..."
                    }
```
Each one has a different crit effect, so we should fill that out for them.